jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/java/org/jbpm/casemgmt/impl/audit/CaseFileDataLogDeleteBuilderImpl.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.casemgmt.impl.audit;
+
+import java.util.Date;
+
+import org.jbpm.process.audit.JPAAuditLogService;
+import org.jbpm.process.audit.query.AbstractAuditDeleteBuilderImpl;
+import org.kie.api.runtime.CommandExecutor;
+import org.kie.internal.query.QueryParameterIdentifiers;
+
+import static org.kie.internal.query.QueryParameterIdentifiers.CASE_FILE_DATA_LOG_LASTMODIFIED;
+
+public class CaseFileDataLogDeleteBuilderImpl extends AbstractAuditDeleteBuilderImpl<CaseFileDataLogDeleteBuilderImpl> {
```

Comment:
Please create a CaseFileDataLogDeleteBuilder interface as we have for other Delete builders and return the interface in the methods.

Reply:
done